spec/controllers/api/v1/application_users_controller_spec.rb

```diff
@@ -46,6 +45,29 @@
     end
   end
 
+  describe "username returns" do
+    it "a single result when username matches" do
+      api_get :username, untrusted_application_token, parameters: { username: 'foo_bb' }
+      expect(response.code).to eq('200')
```

Comment:
This should fail based on the above because `foo_bb` is not an application user for the `untrusted_application`.